ngraph/core/src/op/mvn.cpp

```diff
@@ -117,3 +119,63 @@ bool op::MVN::visit_attributes(AttributeVisitor& visitor)
     visitor.on_attribute("reduction_axes", m_reduction_axes);
     return true;
 }
+
+// ------------------------------ V6 ------------------------------
+
+namespace ngraph
+{
+    template <>
+    NGRAPH_API EnumNames<op::MVNEpsMode>& EnumNames<op::MVNEpsMode>::get()
+    {
+        static auto enum_names =
+            EnumNames<op::MVNEpsMode>("op::MVNEpsMode",
+                                      {{"OUTSIDE_SQRT", op::MVNEpsMode::OUTSIDE_SQRT},
+                                       {"INSIDE_SQRT", op::MVNEpsMode::INSIDE_SQRT}});
+        return enum_names;
+    }
+
+    constexpr DiscreteTypeInfo AttributeAdapter<op::MVNEpsMode>::type_info;
+
+    std::ostream& op::operator<<(std::ostream& s, const op::MVNEpsMode& type)
+    {
+        return s << as_string(type);
+    }
+} // namespace ngraph
+
+NGRAPH_RTTI_DEFINITION(op::v6::MVN, "MVN", 6);
+
+op::v6::MVN::MVN(const Output<Node>& data,
+                 const Output<Node>& reduction_axes,
+                 bool normalize_variance,
+                 double eps,
+                 MVNEpsMode eps_mode)
+    : Op({data, reduction_axes})
+    , m_eps{eps}
+    , m_normalize_variance{normalize_variance}
+    , m_eps_mode{eps_mode}
+{
+    constructor_validate_and_infer_types();
+}
+
+void op::v6::MVN::validate_and_infer_types()
+{
+    set_output_type(0, get_input_element_type(0), get_input_partial_shape(0));
```

Comment:
I missed some checks here: 1. `data` rank >= length(axes); 2. axes is 1d tensor. In more cases axes is a constant so we can check their values as well